R/build.r

```diff
@@ -36,6 +41,20 @@
 #' url: http://pkgdown.r-lib.org
 #' ```
 #'
+#' `deploy` optionally specifies further mechanisms for allowing pkgdown sites
+#'  to link to your package, if your website isn't publically reachable at build-time.
+#'  * To enable autolinking to your privately hosted packag, add a `deploy` field,
+#'    and set its `install_metadata` flag to `true`
+#'    ```yaml
+#'    deploy:
+#'      install_metadata: true
+#'    ```
+#'    * `install_metadata` controls whether a copy of `pkgdown.yml` should be
```

Comment:
I think this would be better as a continuation of the previous bullet, not a new one.

Reply:
resolved by 334c53806242dfb5ef33dfa8b0c7687866c43a35